.github/scripts/check_api.sh

```diff
@@ -366,5 +366,27 @@ test $(echo $req_bulk_applications | jq '.assessments[] | select(.applicationId
 test $(echo $req_bulk_applications | jq '.assessments[] | select(.applicationId == 12) | .status ') = '"STARTED"'
 test $(echo $req_bulk_applications | jq '.assessments[] | select(.applicationId == 325100) | .status ') = '"STARTED"'
 
+echo
+echo
+echo '21 >>> Bulk delete assessments passing in a list of applicationIds and receive 204 as return code'
```

Comment:
I think it's worth adding some more tests to check the transactional endpoint.
It could be something like:

- test 21 tries to delete with 3 application IDs: `applicationSource`, `applicationTarget`, a random one for an not existing assessment and checks it fails
- test 22 checks `applicationSource` assessement is still there
- test 23 checks `applicationSource` applicationTarget is still there
- test 24 deletes providing 2 application IDs: `applicationSource`, `applicationTarget` (corresponding to test 21 in current PR)
- test 25 checks `applicationSource` assessement has been deleted (corresponding to test 22 in current PR)
- test 26 checks `applicationTarget` assessement has been deleted